### PR TITLE
Add retry interval option to create kustomization command

### PR DESCRIPTION
**Pull request by matheuscscp**

### CODE REVIEW

File: cmd/flux/create_kustomization.go [hunk 1]
```diff
@@ -116,6 +117,7 @@ func init() {
 	createKsCmd.Flags().StringVar(&kustomizationArgs.targetNamespace, "target-namespace", "", "overrides the namespace of all Kustomization objects reconciled by this Kustomization")
 	createKsCmd.Flags().StringVar(&kustomizationArgs.kubeConfigSecretRef, "kubeconfig-secret-ref", "", "the name of the Kubernetes Secret that contains a key with the kubeconfig file for connecting to a remote cluster")
 	createKsCmd.Flags().MarkDeprecated("validation", "this arg is no longer used, all resources are validated using server-side apply dry-run")
+	createKsCmd.Flags().DurationVar(&kustomizationArgs.retryInterval, "retry-interval", -1, "the interval at which to retry a previously failed reconciliation (defaults to opt-out)")
```
Comment (stefanprodan) [inline cmd/flux/create_kustomization.go:120]:
```suggestion
	createKsCmd.Flags().DurationVar(&kustomizationArgs.retryInterval, "retry-interval", 0, "the interval at which to retry a previously failed reconciliation")
```

File: cmd/flux/create_kustomization.go [hunk 2]
```diff
@@ -238,6 +240,10 @@ func createKsCmdRun(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	if kustomizationArgs.retryInterval >= 0 {
```
Comment (stefanprodan) [inline cmd/flux/create_kustomization.go:243]:
```suggestion
	if kustomizationArgs.retryInterval > 0 {
```

**Comment by stefanprodan** [comment]
@matheuscscp can you please squash all commits and rebase with upstream main please.

**Comment by matheuscscp** [comment]
> @matheuscscp can you please squash all commits and rebase with upstream main please.

done :ok_hand:

**Pull request merged**